src/pages/Main/Quizzes.jsx

```diff
@@ -29,11 +34,21 @@ export const Quizzes = ({ changeFilter, activeFilters, searchValue }) => {
       try {
         await addFavorite(quizId);
         setFavoritesIds((prevFavoritesIds) => [...prevFavoritesIds, quizId]);
+        setSnackbar({
+          isOpened: true,
+          severity: 'success',
+          message: 'Quiz added to favorites.',
+        });
       } catch (error) {
-        console.error('Error adding to favorites:', error);
+        setSnackbar({
+          isOpened: true,
+          severity: 'error',
```

Comment:
These severities would be easiest to maintain as constants. It's easy to misspell a string and cause an error. If we misspell a constant our IDE will tell us immediately. The constants could look like :

```jsx
severities.ERROR
```

Reply:
Updated this in my next PR